servers/services/src/main/java/org/projectnessie/services/rest/TreeResource.java

```diff
@@ -69,6 +78,18 @@
 public class TreeResource extends BaseResource implements TreeApi {
 
   private static final int MAX_COMMIT_LOG_ENTRIES = 250;
+  private static final ScriptHost SCRIPT_HOST = ScriptHost.newBuilder().build();
```

Comment:
It feels to me like the CEL specific stuff should be in a separate class. Keeps versioning of API simpler if CEL isn't inside imho.

Reply:
+1 on that

Reply:
moved stuff out